README.markdown

```diff
@@ -386,6 +386,10 @@ An optional Lua table can be specified as the last argument to this method to sp
     Specifies whether to perform SSL certificate verification during the
 SSL handshake if the `wss://` scheme is used.
 
+* `headers`
+
+    Specifies custom headers to be sent in the handshake request.
```

Comment:
We should document the structure of this Lua table. As noted below, there could be various different table structures. Giving a code example should be sufficient here.

Reply:
Fixed.